### PR TITLE
feat: update mainnet implementation addresses

### PR DESCRIPTION
**Pull request by ChaoticWalrus**
additionally, add the EigenLayerBeaconOracle to table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +135,7 @@ The current testnet deployment is on holesky, and is from our M2 beta release. Y
...
 | Slasher | [`Slasher`](https://github.com/Layr-Labs/eigenlayer-contracts/blob/testnet-holesky/src/contracts/core/Slasher.sol) | [`0xcAe7...6e7C`](https://holesky.etherscan.io/address/0xcAe751b75833ef09627549868A04E32679386e7C) | [`0x9971...345A`](https://holesky.etherscan.io/address/0x99715D255E34a39bE9943b82F281CA734bcF345A) | Proxy: [OpenZeppelin TUP@4.7.1](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.7.1/contracts/proxy/transparent/TransparentUpgradeableProxy.sol) |
+| EigenLayerBeaconOracle | [Succinct: `EigenLayerBeaconOracle.sol`](https://github.com/succinctlabs/eigenlayer-beacon-oracle/blob/766c8c02030e8c62de5f2dc9934fae389ac2c389/contracts/src/EigenLayerBeaconOracle.sol) | - | [`0x4C11...8f25`](https://holesky.etherscan.io/address/0x4C116BB629bff7A8373c2378bBd919f8349B8f25) | |
```
Comment (wadealexc) [inline README.md:138]: We already have this one in this table!

Comment (wadealexc) [reply README.md:138]: Although yours links to a specific commit which is probably better.

**Review by wadealexc — COMMENTED** [review]
Main thing I'm noticing - the commit hash for the mainnet contracts needs to be updated

**Review by wadealexc — APPROVED** [review]
LGTM aside from the 2 things above ^

File: README.md (hunk 1 above)
Comment (ChaoticWalrus) [reply README.md:138]: ah, my bad, will fix

Comment (ChaoticWalrus) [reply README.md]: got rid of the duplicate entry here https://github.com/Layr-Labs/eigenlayer-contracts/pull/497/commits/79f21bfa07b916b72319ec294d7c021787e94497

**Comment by ChaoticWalrus** [comment]
> Main thing I'm noticing - the commit hash for the mainnet contracts needs to be updated

fixed the commit hash in https://github.com/Layr-Labs/eigenlayer-contracts/pull/497/commits/cc7b97213f1d7b5ef01bff515448e7a9214ca079

**Review by 8sunyuan — APPROVED** [review]
Mainnet addresses look good, just need someone to check my Holesky additions

**Pull request merged**